testharness/src/test/java/bio/terra/policy/service/policy/MutatorTest.java

```diff
@@ -34,29 +36,38 @@ void groupConstraintTest() throws Exception {
     var mcgroupPolicy = new PolicyInput(TERRA, GROUP_CONSTRAINT, buildMultimap(GROUP, MCGROUP));
     var mcyugroupPolicy =
         new PolicyInput(TERRA, GROUP_CONSTRAINT, buildMultimap(GROUP, MCGROUP, YUGROUP));
+    var mcyuddgroupPolicy =
+        new PolicyInput(TERRA, GROUP_CONSTRAINT, buildMultimap(GROUP, MCGROUP, YUGROUP, DDGROUP));
 
     PolicyInput ddmc = policyGroup.combine(ddgroupPolicy, mcgroupPolicy);
 
     // Simple case
```

Comment:
can you split the cases to different test?

Reply:
Why? I'm not seeing the problem.

Reply:
https://engdoc.corp.google.com/eng/doc/tott/episodes/520.md?cl=head

Reply:
It makes no sense to me to break up this test. Will it really help quality or test readability to split this into 6 tests?

Reply:
My 2 cents - I don't treat https://engdoc.corp.google.com/eng/doc/tott/episodes/520.md?cl=head as a hard-and-fast rule. Sometimes it's easier to have tests do a flow, rather than one thing. Also, because these tests do many things, I don't think it makes sense to put the outcome in the name. (That's another rule I don't treat as a hard-and-fast rule.)

Reply:
it's true that testing by case and listing outcome goes hand-in-hand.  i guess it's my OCD from my previous experience to see small tests. imo it improves readibility by seeing the outcome in the test. 

it's just my opinion.